docs/jwst/outlier_detection/outlier_detection.rst

```diff
@@ -0,0 +1,33 @@
+.. _outlier_detection_:
+
+
+Python Interface to OutlierDetection: OutlierDetection()
+=========================================================
+
+This module serves as the interface for applying outlier_detection to direct
+image observations, like those taken with NIRCam.  The code implements the
+basic outlier detection algorithm used with HST data, as adapted to JWST.
+
+Specifically, this routine performs the following operations:
+
+* Extract parameter settings from input model and merge them with any user-provided values
+
+  - If resampling has been turned on for input CubeModel observations, convert the input CubeModel data into a ModelContainer.
+
+* Resamples all input images into grouped observation mosaics.
```

Comment:
Maybe include a reference (link) to an explanation of what is meant by "grouped observation mosaics" (i.e. NIRCam multiple detector images all from a single exposure)